src/core/renderController.ts

```diff
@@ -18,10 +18,10 @@ module Plottable {
    * ```
    */
   export module RenderController {
-    var _componentsNeedingRender = new Utils.Set<Component>();
-    var _componentsNeedingComputeLayout = new Utils.Set<Component>();
-    var _animationRequested = false;
-    var _isCurrentlyFlushing = false;
+    var componentsNeedingRender = new Utils.Set<Component>();
+    var componentsNeedingComputeLayout = new Utils.Set<Component>();
+    var animationRequested = false;
+    var isCurrentlyFlushing = false;
```

Comment:
I think it is more consistent with the rest of the rest of our codebase. Underscored always means not exported, why are these more special?